docs/userguide/networking/dockernetworks.md

```diff
@@ -404,16 +404,27 @@ provision the hosts are with Docker Machine.
 
 ![Engine on each host](images/engine_on_net.png)
 
+It is important to make sure that the following ports are open between each 
```

Comment:
I'd use the imperative. 

You should open the following ports between each of your hosts.

...table..

You key-value store service may require additional open ports.  Check you vendor's documentation and open the appropriate ports.